src/sync/impl/sync_file.hpp

```diff
@@ -111,8 +107,12 @@ class SyncFileManager {
     }
 
 private:
+    // Denotes the root path for any mongodb-realm app for the passed in `base_path`.
+    // Expected to be `base_path` + "mongodb-realm/".
+    const std::string m_root_sync_path;
+    // Denotes the base path for the mongodb-realm app associated with this sync manager.
```

Comment:
Maybe it is just me, but the wording of these comments seems to indicate the naming of the variables should be switched?

Reply:
Hmm... I think the `base_path` parameter in the constructor makes in confusing. You are probably correct, but it does mean `m_base_path` becomes moot.